### PR TITLE
fix(feature-activation): add missing signal bits on get_block_template

### PR DESCRIPTION
**Pull request by glevco**
Depends on https://github.com/HathorNetwork/python-hathorlib/pull/34

### Acceptance Criteria

- Add `signal_bits` on `get_block_template()`
- Bump package to v0.5.1

### CODE REVIEW

File: hathorlib/client.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ async def get_block_template(self, address: Optional[str] = None) -> BlockTempla
 
             # Build block.
             blk = Block()
+            blk.signal_bits = data['signal_bits']
```
Comment (msbrogli) [inline hathorlib/client.py:105]: Should we add a test that would fail without this change?

**Review by msbrogli — APPROVED** [review]

**Review by pedroferreira1 — APPROVED** [review]

File: hathorlib/client.py [hunk 2]
```diff
@@ -85,37 +85,38 @@ async def get_block_template(self, address: Optional[str] = None) -> BlockTempla
...
-
-            do = data['outputs'][0]
-            txout = TxOutput(
-                value=do['value'],
-                token_data=0,
-                script=b'',
-            )
-            blk.outputs = [txout]
-            return BlockTemplate(data=bytes(blk), height=height)
+        resp = await self._session.get(self._get_url('get_block_template'), params=params)
```
Comment (glevco) [inline hathorlib/client.py:88]: I had to change this for it to work with tests, but the behavior should be the same. This new format is just like in the `push_tx_or_block()` method in this class, which we do have tests for.

File: hathorlib/client.py (hunk 1 above)
Comment (glevco) [reply hathorlib/client.py]: Done in https://github.com/HathorNetwork/python-hathorlib/pull/32/commits/8988601cd1291bebe3bd2bf4a79ad0c3886f8a16

**Review by pedroferreira1 — APPROVED** [review]

File: hathorlib/client.py (hunk 1 above)
Comment (msbrogli) [reply hathorlib/client.py]: Thanks!

**Review by msbrogli — APPROVED** [review]

**Pull request merged**